pycqed/measurement/waveform_control/pulsar.py

```diff
@@ -189,8 +189,20 @@ def start(self):
             for AWG in self.used_AWGs():
                 if AWG != self.master_AWG():
                     self._start_AWG(AWG)
-            time.sleep(0.2)  # wait 0.2 second for all other awg-s to start
-            self._start_AWG(self.master_AWG())
+            tstart = time.time()
+            for AWG in self.used_AWGs():
+                if AWG != self.master_AWG():
```

Comment:
A comment here indicating why the master should only be started once the rest is already running would be good. 

A similar comment should be added in line 204.